rpcd/playbooks/verify-maas.yml

```diff
@@ -18,9 +18,12 @@
   user: root
   tasks:
     - name: "Verify Checks & Alarms are registered"
-      script: >
-        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created
-        --entity {{ inventory_hostname }}
+      shell: |
+        {% if maas_venv_enabled | bool %}
+        . {{ maas_venv_bin }}/activate
+        {% endif %}
+        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
```

Comment:
@Frank-ZhangXin so virtualenvs are clever little things. Using `activate` is a short cut for *convenience* not necessity. The main important thing is that the `python` executable in your virtualenv's `bin` directory knows where to look for module imports. That means that you an do

```
myvirtualenv/bin/python some_script.py
```

And if `some_script.py`'s dependencies were installed into that directory, it will Just Work. 

I assume the dependencies are what you mean by plugins.